src/helpers/notifications/Notify.js

```diff
@@ -0,0 +1,39 @@
+import models from '../../models';
+import sendMail from '../Mail/sendMail';
+import eventEmitter from './EventEmitter';
+
+const { Notification, Opt, User } = models;
+
+const notify = async (data) => {
+  let inAppNotification = {};
+  let emailNotification = {};
+
+  const { resource, user, inAppMessage, emailMessage } = data;
+  const optedin = await Opt.findAll({ where: { userId: user.followed } });
```

Comment:
You a try...catch block to handle errors too

Reply:
I didn't added this because what if users don't follow anyone, it will stop the user to create articles and article creation should not be stopped if I face error in notifications.